recipes/pycairo/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1.10.0" %}
+{% set checksum = "9bca41bf9974cff78b9c9b677feacd0c413e79b16cf31ab5d3a3de4e9c48d6db" %}
+{% set checktype = "sha256" %}
+
+package:
+    name: pycairo
+    version: {{ version }}
+
+source:
+    fn: master.zip
+    url: https://github.com/Jactry/pycairo/archive/master.zip
```

Comment:
I changed the repository move from the official pycairo to a github repo because on the official website the last update of the package date from 2011. The official package has issues regarding the install with waf that hasn't been fixed since 2011.